core-util-watcher/index.js

```diff
@@ -0,0 +1,62 @@
+#! /usr/local/bin/node --trace-warnings
+
+// For the package whose source code is at
+// https://github.com/ethereum-optimism/optimism/blob/develop/packages/core-utils/src/watcher.ts
+
+const Watcher = require('@eth-optimism/watcher').Watcher
+const JsonRpcProvider = require('@ethersproject/providers').JsonRpcProvider
+
+const infuraID = "<< PUT YOUR OWN VALUE HERE >>"
+const l1ToL2_L1TxHash = '0x65d50ed9d1edd7f20e97229667b5fc892fb23b128c8baca960c47c74fde9d4ed'
+const l2ToL1_L2TxHash = "0xa018b12df969c0f65aec3c3d08c94332525a309cdec38206eef74319c1d1dd30"
+
+const kovanWatcher = new Watcher({
+    l1: {
+      provider: new JsonRpcProvider(`https://kovan.infura.io/v3/${infuraID}`),
+      messengerAddress: '0x4361d0F75A0186C05f971c566dC6bEa5957483fD'
+    },
+    l2: {
+      provider: new JsonRpcProvider(`https://kovan.optimism.io`),
+      messengerAddress: '0x4200000000000000000000000000000000000007'
+    }
+  })
+
+
+  const mainnetWatcher = new Watcher({
+    l1: {
+      provider: new JsonRpcProvider(`https://mainnnet.infura.io/${infuraID}`),
+      // On Kovan, the value for mainnet is
+      messengerAddress: '0x25ace71c97B33Cc4729CF772ae268934F7ab5fA1'
+    },
+    l2: {
+      provider: new JsonRpcProvider(`https://mainnet.optimism.io`),
+      messengerAddress: '0x4200000000000000000000000000000000000007'
+    }
+  })  
+
+
+const watcher = kovanWatcher;   // mainnetWatcher
```

Comment:
I tried running the example for a mainnet transaction `0x122abee5d2f41da16a40e7edd0c5460d8f50683a46e1765fee2a8a8eb8b01cc2` but it throws with 
```
(node:33028) UnhandledPromiseRejectionWarning: Error: could not detect network (event="noNetwork", code=NETWORK_ERROR, version=providers/5.4.3)
```

Reply:
Fixed, thank you.

Reply:
Solved it, my bad.